README.md

```diff
@@ -60,57 +60,20 @@ Download the pre-compiled package for your operating system.  Install required d
   If the imaging install script reports errors in creating directories (due to /data/ mount permissions), review and manually execute `mkdir/chmod/chown` commands starting at [imaging_install.sh:L97](https://github.com/aces/Loris-MRI/blob/master/imaging_install.sh#L97)
 
   Note: The installer will allow Apache to write to the /data/ directories by adding user lorisadmin to the Apache linux group.  To ensure this change takes effect, log out and log back into your terminal session before running the imaging pipeline.
-The installer will also set Apache group ownership of certain /data/ subdirectories.  
+The installer will also set Apache group ownership of certain /data/ subdirectories.
 
-#### 4. Configure paths and environment
-
-  To help ensure Apache-writability, verify that your environment file contains the following line:
+  After having run the install script, it is possible to verify that certain
+    fields were correctly populated by `imaging_install.sh` (see
+    [section 2.4](02-Install.md#2.4-post-installation-checks)).
 
-   ```bash 
-   umask 0002
-   ```
+#### 4. Configure paths and environment
 
-   Ensure that /home/lorisadmin/.bashrc includes the statement: 
+   Ensure that /home/lorisadmin/.bashrc includes the statement:
 
    ```source /data/$projectname/bin/mri/environment```
 
    Then source the .bashrc file.   
 
-#### 5. Set up MINC utilities for BrainBrowser visualization
-
-To ensure that BrainBrowser can load MINC images, the MINC toolkit must be accessible to the main LORIS codebase.
-(If the Loris-MRI codebase is installed on a separate machine, ensure the MINC toolkit is installed in both locations.)
-
-Ensure the _project/config.xml_ file (in the main LORIS codebase) contains the following tagset, specifying the MINC toolkit path local to the main LORIS codebase (/opt/minc/ in this example):
-
-   ```xml
-   <!-- MINC TOOLS PATH -->
-   <MINCToolsPath>/opt/minc/</MINCToolsPath>
-   ```
-
-#### 6. Verify filesystem permissions 
-
-Ensure that permissions on /data/$projectname and /data/incoming and their subdirectories are set such that lorisadmin and the Apache linux user can read, write _and_ execute all contents.
-
-The following must be recursively owned by the lorisadmin user and by Apache group:
-
-   ```bash
-   /data/$projectname/data/ 
-   /data/$projectname/bin/mri/
-   /data/incoming/
-   /data/$projectname/bin/mri/dicom-archive/.loris_mri/prod
-   ```
-#### 7. Verify Configuration module settings for Imaging Pipeline
-  
-In the LORIS front-end, under the Admin menu, go to the `Config` module.  Under the section `Imaging Pipeline`, verify/set the following config settings: 
- * `Loris-MRI Data Directory`
- * `Study Name`
- * `User to notify when executing the pipeline`
- * `Full path to get_dicom_info.pl script`
- * `Path to Tarchives`
-
-Click 'Submit' at the end of the Configuration page to save any changes. 
-
 <br>
 Installation complete.
 
```

Comment:
@cmadjar 

after this line, can we link to the Post-Installation checks you moved to the docs/ section 2.4. What do you think?

Reply:
@MounaSafiHarab Ooops... Yes, definitely. I think I was about to and then got distracted :S. Thanks!

Reply:
we need to eventually have the latest README file replacing the one here (it is a little outdated now)